pkg/kapis/iam/v1alpha2/register.go

```diff
@@ -44,7 +44,7 @@ func AddToContainer(c *restful.Container, k8sClient k8s.Client, factory informer
 	handler := newIAMHandler(k8sClient, factory, ldapClient, cacheClient, options)
 
 	// implemented by create CRD object.
-	//ws.Route(ws.POST("/users"))
+	//ws.Route(ws.POST("/users").To(handler.CreateUser))
```

Comment:
What's the intention of exposing users api on ks-apiserver, since we can do it directly through kube-apiserver?